File: PHP Word Examples/ConvertApiDemo/convertToHtmlInRequest.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\ConvertApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ConvertApi($configuration);

$inputFile = "D:/data/ConvertToHtmlInRequest.docx";
$password = null;
$result=$apiInstance->convertToHtmlInRequest($inputFile, $password);

File: Python Word Examples/ConvertApiDemo/convertToHtmlInRequest.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.convert_api.ConvertApi(configuration)

file_name = "D:/input/ConvertToHtmlInRequest.docx"
result = api.convert_to_html_in_request(file_name)
